tests/ashley/test_models_lticontext.py

```diff
@@ -112,3 +112,50 @@ def test_sync_user_groups(self):
             ],
             list(user.groups.values_list("name", flat=True)),
         )
+
+    def test_get_group_role_name(self):
+        """Test get_group_role_name that returns the name of the role with the specific pattern"""
```

Comment:
I would say we usually use imperative form for test docstrings (maybe it could go to the handle book if everyone agrees with this):
```suggestion
        """get_group_role_name should return the name of the role with the specific pattern"""
```